lib/image_render.dart

```diff
@@ -34,21 +34,21 @@ ImageSourceMatcher networkSourceMatcher({
     };
 
 ImageSourceMatcher assetUriMatcher() => (attributes, element) =>
-    _src(attributes) != null && _src(attributes).startsWith("asset:");
+    _src(attributes) != null && _src(attributes)!.startsWith("asset:");
 
 typedef ImageRender = Widget Function(
```

Comment:
This is a bug. It _should_ allow returning `null` here, so the signature should be 
```
typedef ImageRender = Widget? Function(
  RenderContext context,
  Map<String, String> attributes,
  dom.Element? element,
);
```

Reply:
Done.